src/Microsoft.AspNetCore.Mvc.Analyzers/SymbolNames.cs

```diff
@@ -5,12 +5,22 @@ namespace Microsoft.AspNetCore.Mvc.Analyzers
 {
     internal static class SymbolNames
     {
-        public const string IHtmlHelperType = "Microsoft.AspNetCore.Mvc.Rendering.IHtmlHelper";
+        public const string AllowAnonymousAttribute = "Microsoft.AspNetCore.Authorization.AllowAnonymousAttribute";
```

Comment:
Curious why static strings? why not static and readonly typeof(..).FullName?

Reply:
The analyzer does not reference Mvc. So `typeof` wouldn't work